great_expectations/rule_based_profiler/util.py

```diff
@@ -299,20 +343,37 @@ def compute_bootstrap_quantiles(
     bootstraps: np.ndarray = np.random.choice(
         metric_values, size=(n_resamples, metric_values.size)
     )
-    lower_quantile = np.mean(
-        np.quantile(
-            bootstraps,
-            q=false_positive_rate / 2,
-            axis=1,
-            interpolation="linear",  # can be omitted ("linear" is default)
-        )
+    lower_quantiles: Union[np.nd_array, Number] = np.quantile(
+        bootstraps,
+        q=false_positive_rate / 2,
+        axis=1,
     )
-    upper_quantile = np.mean(
-        np.quantile(
-            bootstraps,
-            q=1.0 - (false_positive_rate / 2),
-            axis=1,
-            interpolation="linear",  # can be omitted ("linear" is default)
-        )
+    lower_quantile_point_estimate: Number = np.mean(lower_quantiles)
+
+    lower_quantile_residuals: Union[np.ndarray, Number] = (
+        lower_quantiles - lower_quantile_point_estimate
+    )
+    lower_quantile_bias: Number = sum(lower_quantile_residuals) / len(lower_quantiles)
+    lower_quantile_bias_corrected_point_estimate: Number = (
+        lower_quantile_point_estimate + lower_quantile_bias
+    )
+
+    upper_quantiles: Union[np.nd_array, Number] = np.quantile(
+        bootstraps,
+        q=1.0 - (false_positive_rate / 2),
+        axis=1,
+    )
+    upper_quantile_point_estimate: Number = np.mean(upper_quantiles)
+
+    upper_quantile_residuals: Union[np.ndarray, Number] = (
+        upper_quantiles - upper_quantile_point_estimate
+    )
+    upper_quantile_bias: Number = sum(upper_quantile_residuals) / len(upper_quantiles)
+    upper_quantile_bias_corrected_point_estimate: Number = (
+        upper_quantile_point_estimate + upper_quantile_bias
```

Comment:
@NathanFarmer Where is this bias correction formula described?  Could you please point me to a resource where I can see the proof of the math involved?  Essentially, the code says (where "^T" denotes vector/matrix transpose):
```
 mean(statistic) + [statistic - mean(statistic)]^T * [1-vector] / N
```
which is to say "mean of statistic, corrected by average deviation between statistic and mean of statistic itself").  I would be curious to see the math the shows the reason that this bias correction method is advantageous.  Thank you!

Reply:
@NathanFarmer I do recall using a different approach.  It may well have been using the quantiles (upper, lower) as the statistics and returning the arithmetic mean of the confidence interval, but I am not sure -- still looking.  I also recall using the `SciPy` bootstrap API to compute all the bootstraps for us by supplying it with the data.  I feel that we should not be doing this ourselves (i.e., we should not be computing the bootstraps ourselves, we should be leveraging the SciPy library for this).  Thanks.

Reply:
This is outdated and this method has been changed. See:
```
    Efron, B., & Tibshirani, R. J. (1993). Estimates of bias. An Introduction to the Bootstrap (pp. 124-130).
        Springer Science and Business Media Dordrecht. DOI 10.1007/978-1-4899-4541-9
```
for the new method.